decidim/decidim
# Fix character counter for the WYSIWYG editor

#### :tophat: What? Why?
The character counter is currently broken for the WYSIWYG editor I believe after the screen reader updates at #9009.

This makes the character counter work again with the WYSIWYG editor.

#### :pushpin: Related Issues
- Related to #9009

#### Testing
- Enable rich text editor for participants
- Create a new proposal
- See that the character counter is working correctly in the view

Note that the "empty" field character counting is a bit buggy in Quill.js (which is the rich text editor). It automatically injects one paragraph to the text with a line break in it, so I believe it is because of this reason why it calculates the "empty" input as 2 characters.
